autodist/kernel/graph_transformer.py

```diff
@@ -161,8 +163,10 @@ def _between_graph_apply(self, multi_gpu_graph_item: GraphItem):
             GraphItem
         """
         new_graph_item = multi_gpu_graph_item
+        new_graph_item.start_loop_optimize()
```

Comment:
`Start_loop_optimize()` sounds weird; maybe change the signature... like `set_mutatle()` etc.